fmriprep/reports/core.py

```diff
@@ -54,10 +78,52 @@ def generate_reports(subject_list, output_dir, run_uuid, config=None, work_dir=N
             import sys
             import traceback
 
+            # Store the list of subjects for which report generation failed
             errors.append(subject_label)
             traceback.print_exception(
                 *sys.exc_info(),
                 file=str(Path(output_dir) / "logs" / f"report-{run_uuid}-{subject_label}.err"),
             )
 
+        if n_ses >= MAX_SES_AGR:
```

Comment:
Should this not be an option for the CLI? (with `MAX_SES_AGR` being the default)

Reply:
Done in https://github.com/nipreps/fmriprep/pull/3191/commits/87db1c9625263e80246988b767ffe9986b0666fe

Reply:
Tested it works